doc/guidelines/ui.md

```diff
@@ -0,0 +1,38 @@
+# UI
+
+## Submitting Data
+
+Data should be always submitted on-the-fly but with debouncing.
+
+## Okay Buttons
+
+If needed, the confirming button should be on the left.
+
+## Gender
+
+Use plural to avoid gender, avoid he/she.
```

Comment:
```suggestion
Use plural to avoid gendered nouns, avoid he/she.
```
Which pronoun should be used insted? They/Them?
Please clarify.

Reply:
"Use plural to avoid gender, i.e. use "we" or "they", avoid "he"/"she"."

6594d65a4681a926c5c0fcac3e1bcf46b4aab632